src/common/utils.js

```diff
@@ -1 +1,31 @@
-// TODO Menu creator object
+import React from 'react'
+import uuid from 'uuid'
+import { Icon, Menu } from 'antd'
+import { Link } from 'react-router-dom'
+
+const { Item, SubMenu } = Menu
+
+// eslint-disable-next-line import/prefer-default-export
+export const MenuCreator = menuListItems =>
+  menuListItems.map(item => {
+    return item.submenu.length ? (
+      <SubMenu
+        key={uuid()}
```

Comment:
i think `shortId` https://www.npmjs.com/package/short-id is better alternative for keys